README.md

```diff
@@ -0,0 +1,9 @@
+## Build
+
+```
+$ ./autogen.sh
```

Comment:
To keep things simple I was avoiding auto-generated files in the source. I've seen enough comments bemoaning what happens if you do to put me off. That doesn't stop us including the results of autogen.sh in a distribution though.

I'm going to change the name to [bootstrap](https://www.sourceware.org/autobook/autobook/autobook_43.html) though.